app/config/default.json

```diff
@@ -1,5 +1,5 @@
 {
   "protocol": "http",
   "host": "localhost",
-  "port": 5001
-}
\ No newline at end of file
+  "port": 3447
```

Comment:
I think this needs to be reverted back to 5001 for routing the transactions through the OpenHIM